.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 /dist
+/**/dist/**.js
```

Comment:
Just wonder how is this being used, i don't see any file generated on a dist folder.